libs/sql-parser/src/test/java/io/crate/sql/IdentifiersTest.java

```diff
@@ -35,7 +35,7 @@ public void test_number_of_keywords() {
         // Either add the new term to `nonReserved` in `SqlBase.4g` or add a breaking changes entry and adapt this test.
         assertThat(
             (int) Identifiers.KEYWORDS.stream().filter(Identifiers.Keyword::isReserved).count())
-            .isEqualTo(96);
+            .isEqualTo(97);
```

Comment:
The keyword shouldn't be reserved, otherwise this is a breaking change

Reply:
but it's on the list here: https://www.postgresql.org/docs/current/sql-keywords-appendix.html
and the change is for 6.0, so I'd rather add it as breaking change, wdyt?

Reply:
I don't think so, I will remove it from reserved. thx!

Reply:
After sleeping on this I think making `current_catalog` reserved might be the better choice.

Main reason is that it is breaking in any case.

Say a user has a table `create table tbl (current_catalog text)` and queries like `select current_catalog from tbl`.
With this change, it would change the `current_catalog` expression to return the value of the function, instead of the column.
If we make it reserved, there's at least an error to make this mistake easier to spot.

Reply:
ok, I agree, but before I'll merge the fix: https://github.com/crate/crate/pull/17994